### ISSUE TITLE
CT unexpected number of particles in saved data set

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/states-of-matter#254]
```
states-of-matter : phet-io-state-fuzz : require.js : run
Uncaught Error: Assertion failed: unexpected number of particles in saved data set
Error: Assertion failed: unexpected number of particles in saved data set
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/assert/js/assert.js?bust=1582602021147:22:13)
    at WaterPhaseStateChanger.loadSavedState (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1582602021302:348:17)
    at WaterPhaseStateChanger.setParticleConfigurationSolid (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/WaterPhaseStateChanger.js?bust=1582602021302:720:12)
    at WaterPhaseStateChanger.setParticleConfigurationForPhase (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1582602021302:73:16)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1582602021302:301:36
    at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/TinyEmitter.js?bust=1582602021302:68:55)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Emitter.js?bust=1582602021302:36:31
    at Emitter.execute (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Action.js?bust=1582602021302:227:20)
    at Emitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Emitter.js?bust=1582602021302:61:21)
    at PhetioStateEngine.setState (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/phet-io/js/PhetioStateEngine.js?bust=1582602021302:189:28)
id: Bayes Chrome
Approximately 2/24/2020, 6:58:04 PM
states-of-matter : phet-io-state-fuzz : require.js : run
Uncaught Error: Assertion failed: unexpected number of particles in saved data set
Error: Assertion failed: unexpected number of particles in saved data set
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/assert/js/assert.js?bust=1582622485981:22:13)
    at WaterPhaseStateChanger.loadSavedState (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1582622486123:348:17)
    at WaterPhaseStateChanger.setParticleConfigurationSolid (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/WaterPhaseStateChanger.js?bust=1582622486123:720:12)
    at WaterPhaseStateChanger.setParticleConfigurationForPhase (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1582622486123:73:16)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1582622486123:301:36
    at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/TinyEmitter.js?bust=1582622486123:68:55)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Emitter.js?bust=1582622486123:36:31
    at Emitter.execute (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Action.js?bust=1582622486123:227:20)
    at Emitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Emitter.js?bust=1582622486123:61:21)
    at PhetioStateEngine.setState (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/phet-io/js/PhetioStateEngine.js?bust=1582622486123:189:28)
id: Bayes Chrome
Approximately 2/24/2020, 6:58:04 PM
states-of-matter : phet-io-tests : assert
9 out of 9 tests passed. 0 failed.

Approximately 2/24/2020, 6:58:04 PM
states-of-matter : phet-io-tests : no-assert
9 out of 9 tests passed. 0 failed.

Approximately 2/24/2020, 6:58:04 PM
```

**Comment by KatieWoe** [comment]
Also in Basics sim:
```
states-of-matter-basics : phet-io-state-fuzz : require.js : run
Uncaught Error: Assertion failed: unexpected number of particles in saved data set
Error: Assertion failed: unexpected number of particles in saved data set
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/assert/js/assert.js?bust=1582600170238:22:13)
    at WaterPhaseStateChanger.loadSavedState (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1582600170376:348:17)
    at WaterPhaseStateChanger.setParticleConfigurationSolid (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/WaterPhaseStateChanger.js?bust=1582600170376:720:12)
    at WaterPhaseStateChanger.setParticleConfigurationForPhase (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1582600170376:73:16)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1582600170376:301:36
    at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/TinyEmitter.js?bust=1582600170376:68:55)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Emitter.js?bust=1582600170376:36:31
    at Emitter.execute (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Action.js?bust=1582600170376:227:20)
    at Emitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Emitter.js?bust=1582600170376:61:21)
    at PhetioStateEngine.setState (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/phet-io/js/PhetioStateEngine.js?bust=1582600170376:189:28)
id: Bayes Chrome
Approximately 2/24/2020, 6:58:04 PM
states-of-matter-basics : phet-io-state-fuzz : require.js : run
Uncaught Error: Assertion failed: unexpected number of particles in saved data set
Error: Assertion failed: unexpected number of particles in saved data set
    at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/assert/js/assert.js?bust=1582614229429:22:13)
    at WaterPhaseStateChanger.loadSavedState (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1582614229576:348:17)
    at WaterPhaseStateChanger.setParticleConfigurationSolid (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/WaterPhaseStateChanger.js?bust=1582614229576:720:12)
    at WaterPhaseStateChanger.setParticleConfigurationForPhase (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/engine/AbstractPhaseStateChanger.js?bust=1582614229576:73:16)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/states-of-matter/js/common/model/MultipleParticleModel.js?bust=1582614229576:301:36
    at TinyEmitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/TinyEmitter.js?bust=1582614229576:68:55)
    at https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Emitter.js?bust=1582614229576:36:31
    at Emitter.execute (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Action.js?bust=1582614229576:227:20)
    at Emitter.emit (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/axon/js/Emitter.js?bust=1582614229576:61:21)
    at PhetioStateEngine.setState (https://bayes.colorado.edu/continuous-testing/snapshot-1582595884495/phet-io/js/PhetioStateEngine.js?bust=1582614229576:189:28)
id: Bayes Chrome
Approximately 2/24/2020, 6:58:04 PM
states-of-matter-basics : phet-io-tests : assert
9 out of 9 tests passed. 0 failed.

Approximately 2/24/2020, 6:58:04 PM
states-of-matter-basics : phet-io-tests : no-assert
9 out of 9 tests passed. 0 failed.

Approximately 2/24/2020, 6:58:04 PM
```

**Comment by jbphet** [comment]
This has been fixed by work done for #252.  Closing.
